### DISCUSSION TITLE
wondering for a new multiplayer battle mode

### DISCUSSION THREAD

**Discussion by SundayLee404** [MrStahlfelge/lightblocks#40] [Ideas]
how will it be if can playing battle mode with freeze.
just like playing tetris effect: connected's zone battle.

**Comment by MrStahlfelge** [comment]
Are you referring to the turn based battle or the server multiplayer mode here?

**Reply by SundayLee404** [reply]
server multiplayer mode

**Reply by MrStahlfelge** [reply]
Yes, good ideas and it is technically possible by changing just the server. However, as pointed out in #41, I am not sure if the server multiplayer will get many players as there are adoption problems

**Reply by SundayLee404** [reply]
Ok
or also make this in Turn-based battle

**Comment by SundayLee404** [comment]
and I pretty want to let Turn-Based battle's type can be choosed between Classic and Modern.
and make another leaderboard for Modern type.
